ci-operator/jobs/devfile/integration-tests/devfile-integration-tests-main-periodics.yaml

```diff
@@ -161,7 +161,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
   cluster: build05
-  cron: 0 2 * * 5
+  cron: 0 2 * * 1-5
```

Comment:
Use old cron configuration

Reply:
fixed in https://github.com/openshift/release/pull/53140/commits/41c10053c310963005662e32116fe5af9a32637b